### PR TITLE
Expose VarDictJava on the command line.

### PR DESCRIPTION
**Pull request by nh13**
Also fixed a bug where VarDictJava would require one more core than needed.

### CODE REVIEW

File: tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala [hunk 1]
```diff
@@ -108,24 +111,32 @@ private class VarDictJava(tumorBam: PathToBam,
...
+      |Tumor-only single-sample variant calling with VarDictJava: https://github.com/AstraZeneca-NGS/VarDictJava
+      |
+      |The output Tumor name will be inferred from the the first read group in the input BAM if not provided.
+    """",
+  group = classOf[Pipelines]
+)
+class VarDictJavaEndToEnd
+(@arg(flag="i", doc="The input tumor BAM file.") tumorBam: PathToBam,
+ @arg(flag="l", doc="The intervals oer which to call variants.") intervals: PathToIntervals,
+ @arg(flag="r", doc="Path to the reference fasta file.")         val ref: PathToFasta,
```
Comment (tfenne) [inline tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala]: Extra spaces.

File: tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala [hunk 2]
```diff
@@ -108,24 +111,32 @@ private class VarDictJava(tumorBam: PathToBam,
...
+      |
+      |Tumor-only single-sample variant calling with VarDictJava: https://github.com/AstraZeneca-NGS/VarDictJava
+      |
+      |The output Tumor name will be inferred from the the first read group in the input BAM if not provided.
+    """",
+  group = classOf[Pipelines]
+)
+class VarDictJavaEndToEnd
+(@arg(flag="i", doc="The input tumor BAM file.") tumorBam: PathToBam,
+ @arg(flag="l", doc="The intervals oer which to call variants.") intervals: PathToIntervals,
```
Comment (tfenne) [inline tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala]: oer -> over

**Review by tfenne — COMMENTED** [review]
The only thing I'm not 100% keen on is the change to take in an interval list and convert it to a BED.  Mostly because I use this in a lot of pipelines, and I like to keep the BED file around for use after the fact.

**Comment by nh13** [comment]
I kept the `bed` option and addressed your comments, merging once it passes.

**Pull request merged**